packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -518,6 +520,22 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
       await idle();
     });
   }
+
+  /// Makes an effort to dismiss the current page with a Material [Scaffold] or
+  /// a [CupertinoPageScaffold].
+  ///
+  /// Will throw an error if there is no scaffold, no back button in the
+  /// scaffolds or the back button are not tappable.
```

Comment:
nit: _is_ not tappable?
nit: I learned recently that we prefer Oxford commas in docs, so "...scaffold, or the back button..."

Reply:
Changed the comments